ansible/couchdb.yml

```diff
@@ -1,6 +1,16 @@
 ---
 # This playbook deploys a CouchDB for Openwhisk.  
 
+- hosts: localhost
```

Comment:
I moved these tasks from `setup.yml` - shouldn't they be here instead of the host hack that was injected there? It is cleaner to separate the host file generation from the couchdb generation. @csantanapr thoughts?